nasa/harmony
# HARMONY-294: Update STAC output to version 1.0.0 and fix geometry

## Jira Issue ID
HARMONY-294

## Description
Updates STAC output to version 1.0.0 and fixes STAC item geometry to have a correct Multipolygon `coordinates` entry.

## Local Test Steps
1. check out this branch and run harmony
2. issue this query
http://localhost:3000/C1234088182-EEDTEST/ogc-api-coverages/1.0.0/collections/blue_var/coverage/rangeset?format=image%2Fpng&granuleID=G1256340898-EEDTEST
4. Wait for the job to complete then go to the STAC item file at
http://localhost:3000/stac/<job-id>/0
5. Verify that the geometry type is Multipolygon and that the 'coordinates' section has four levels of arrays.
6. Optionally paste the STAC item JSON [here](https://staclint.com/) to validate it.

## PR Acceptance Checklist
